app/config/tstvhq/rooms.json

```diff
@@ -283,13 +274,13 @@
         {
             "id": "aviary",
             "name": "Aviary",
-            "type": "IFRAME",
+            "type": "JITSI",
```

Comment:
you changed this to the `jitsi` room type but left the `iframeOptions`, so it's unclear what the intention is here. same with sleeping porch